packages/database/prisma/schema.prisma

```diff
@@ -7,6 +7,75 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
+model Account {
+  id                       String   @id @default(cuid()) @map("_id")
+  userId                   String   @map("user_id")
+  type                     String
+  provider                 String
+  providerAccountId        String   @map("provider_account_id")
+  refresh_token            String?  @db.String
+  refresh_token_expires_in Int?     @db.Int
+  access_token             String?  @db.String
+  expires_at               Int?
+  token_type               String?
+  scope                    String?
+  id_token                 String?  @db.String
+  session_state            String?
+  anonymous                Boolean
+  created                  Boolean
+  createdAt                DateTime @db.Date
+  firstName                String?
+  lastName                 String?
+  linkedin                 String
+  location                 String
+  /// Could not determine type: the field only had null or empty values in the sample set.
+  offers                   Json?
+  pfp                      String
+  pipeline                 Json?
+  position                 String
+  /// Multiple data types found: String: 86.5%, String (ObjectId): 13.5% out of 89 sampled entries
```

Comment:
Yeah looks like we have some inconsistencies with how we made the Account --> School relationship. I think the new standard we set from last time was using ObjectIds instead of literal string names.

Reply:
Yeah I'll add the ObjectId decorator